src/components/kyb/ComplianceStep.tsx

```
@@ -0,0 +1,292 @@
+  const [showValidation, setShowValidation] = useState(false);
+
+  const handleInputChange = (field: keyof BusinessData, value: string | boolean) => {
+    updateData({ [field]: value });
+    
+    // Clear error when user checks the box
+    if (value === true && errors[field as keyof ValidationErrors]) {
+      setErrors(prev => ({ ...prev, [field]: undefined }));
+    }
+  };
+
+  const validateRequiredDeclarations = () => {
```

ESLint

[error] 'validateRequiredDeclarations' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)